Description: Remove some unused features
 The unstable, channel-api, quick_test and check_asm features are unused. The features
 desync_finder, dump_ivf and dump_lookahead_data are only useful for internal development.
Author: Sebastian Ramacher <sramacher@debian.org>
Last-Update: 2022-08-21
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -45,11 +45,6 @@
 bench = false
 required-features = ["binaries"]
 
-[[bin]]
-name = "rav1e-ch"
-bench = false
-required-features = ["binaries", "channel-api", "unstable"]
-
 [dependencies.aom-sys]
 version = "0.3.0"
 optional = true
@@ -65,17 +60,9 @@
 optional = true
 default-features = false
 
-[dependencies.backtrace]
-version = "0.3"
-optional = true
-
 [dependencies.bitstream-io]
 version = "1"
 
-[dependencies.byteorder]
-version = "1.3.2"
-optional = true
-
 [dependencies.cfg-if]
 version = "1.0"
 
@@ -89,10 +76,6 @@
 version = "0.15"
 optional = true
 
-[dependencies.crossbeam]
-version = "0.8"
-optional = true
-
 [dependencies.dav1d-sys]
 version = "0.5"
 optional = true
@@ -101,12 +84,6 @@
 version = "0.6"
 optional = true
 
-[dependencies.image]
-version = "0.23"
-features = ["png"]
-optional = true
-default-features = false
-
 [dependencies.itertools]
 version = "0.10"
 
@@ -200,19 +177,12 @@
 bench = []
 binaries = ["ivf", "y4m", "clap", "scan_fmt", "fern", "console", "av-metrics"]
 capi = []
-channel-api = ["crossbeam"]
-check_asm = []
 decode_test = ["aom-sys"]
 decode_test_dav1d = ["dav1d-sys"]
 default = ["binaries", "asm", "signal_support"]
-desync_finder = ["backtrace"]
-dump_ivf = ["ivf"]
-dump_lookahead_data = ["byteorder", "image"]
-quick_test = []
 scenechange = []
 serialize = ["serde", "toml", "v_frame/serialize", "arrayvec/serde"]
 signal_support = ["signal-hook"]
-unstable = []
 [target."cfg(any(decode_test, decode_test_dav1d))".dependencies.system-deps]
 version = "6"
 
